lnwallet/channel.go

```diff
@@ -2012,7 +2041,11 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 		payDesc, err := lc.remoteLogUpdateToPayDesc(
 			&logUpdate, lc.localUpdateLog, localCommitmentHeight,
 		)
-		if err != nil {
+		if err == errCancelledAdd {
```

Comment:
This really is not an expected error, so we should at least log it.

Reply:
Logging error now